crates/runtime/wit/host.wit

```diff
@@ -6,13 +6,18 @@ world interfaces {
     import wasi:keyvalue/atomics@0.2.0-draft;
     import wasi:keyvalue/store@0.2.0-draft;
     import wasi:keyvalue/batch@0.2.0-draft;
-    import wasi:logging/logging;
+    import wasi:logging/logging@0.1.0-draft;
+    
```

Comment:
Could we import both here instead? (i.e. use a versioned *and* unversioned identifier)
If not, perhaps we can `include old-interfaces` instead of generating bindings twice?

Reply:
I couldn't get the bindings to generate both if I included it in the same world. That was the first thing I tried